plugins/USBnull/USB.h

```diff
@@ -21,7 +21,6 @@
 #define USBdefs
 #include "PS2Edefs.h"
 #include "PS2Eext.h"
-
```

Comment:
This was fine as it was, not sure why you removed it, and changing the code in USB plugin.